arch/risc-v/src/Makefile

```diff
@@ -126,6 +126,9 @@ $(AOBJS) $(UAOBJS) $(HEAD_OBJ): %$(OBJEXT): %.S
 $(COBJS) $(UCOBJS): %$(OBJEXT): %.c
 	$(call COMPILE, $<, $@)
 
+crt0$(OBJEXT): %$(OBJEXT): %.c
```

Comment:
Separate rule because adding the file to STARTUP_OBJS fails, the list then contains one asm and one c file and the rule for STARTUP_OBJS gets confused..